semanticz-connector-graphdb/src/main/java/zone/cogni/semanticz/connectors/graphdb/GraphDBSparqlService.java

```diff
@@ -42,6 +41,7 @@
 import java.nio.charset.StandardCharsets;
 import java.util.function.Function;
 
+import static zone.cogni.semanticz.connectors.utils.Constants.CONTENT_TYPE;
```

Comment:
hmm. I would not have done this. it is probably easier to backport fixes if both branches are as similar as possible?

Reply:
You mean try to stay with HttpClient 4?

Reply:
HttpClient 4 works OK, for some reason I was getting some issues before which I can't replicate anymore ... my bad...

Reply:
No, just more the split of actual upgrade vs cleaning up code.